### ISSUE TITLE
Travis CI no more building py34-dj19-cms34

### ISSUE THREAD

**Issue by fp4code** [divio/djangocms-snippet#42]
Version 1.9.2 has been built smoothely 10 month ago: https://travis-ci.org/divio/djangocms-snippet/builds/178035225

Now I got problems trying to rebuild exactly the same code: https://travis-ci.org/fp4code/djangocms-snippet/builds/278143612
(Only one change: the text file `this-file-is-only-a-travis-starter.txt` was added to fire Travis on my Fork).

Please, can you ask Travis to rebuild master code on this page: https://travis-ci.org/divio/djangocms-snippet/builds/178035225 ?

**Comment by fp4code** [comment]
Local simulation, step-by-step

```
virtualenv env
source env/bin/activate
python --version # Python 2.7.13
pip --version # pip 9.0.1
git clone https://github.com/divio/djangocms-snippet.git
cd djangocms-snippet
git log -1 # commit 0bf25960 Release version 1.9.2
pip install tox coverage
tox -e py34-dj19-cms34
```
```
  File "djangocms_snippet/templatetags/snippet_tags.py", line 5, in <module>
    import six
ImportError: No module named 'six'
```

The problem is that `six` package is no more installed by default for `python3`.

**Comment by fp4code** [comment]
Solved by pull request #43
